tools/Modules/TestFx-Tasks.psd1

```diff
@@ -0,0 +1,126 @@
+#
+# Module manifest for module 'PSGet_TestFx-Tasks'
+#
+# Generated by: ShahAbhijeet
+#
+# Generated on: 11/18/2016
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
```

Comment:
@markcowl almost every time when people try to use our test framework they ask how can I create service principal. So able to create service principal when someone is trying to use our module to create connection string is something that would be beneficial.
This is again part of the various tasks that would help our RPs to use our test infrastructure. There is not task item for this particular PR.